src/components/Shortcuts/index.jsx

```diff
@@ -1,9 +1,14 @@
 import React from 'react'
+import flag from 'cozy-flags'
 
 import { ShortcutsView } from './ShortcutsView'
+import { SectionsView } from 'components/Sections/SectionsView'
 
-export const Shortcuts = ({ shortcutsDirectories }) => (
-  <ShortcutsView shortcutsDirectories={shortcutsDirectories} />
-)
+export const Shortcuts = ({ shortcutsDirectories }) =>
+  flag('home.sections') ? (
+    <SectionsView data={shortcutsDirectories} />
```

Comment:
Maybe we can avoid creating the `SectionsView` and directly make the map here 🤔

Reply:
agreed ✅